etc/ui-config/mh_default_org/editor/editor-settings.toml

```diff
@@ -0,0 +1,26 @@
+### Ungrouped values, used to make GET-Parameters easier to write
+### Values are assigned here only for debugging purposes
+# Id of the event that the editor should open
+#mediaPackageId = '27cd7156-fda6-4b31-aab5-d56833012caf'
+
+#### Values pertaining to accessing opencast
+[opencast]
+# URL of the opencast server on which we are editing events.
+# Should be deleted when deployed in Opencast, as the editor can then
+# figure out the correct URL by itself.
+#url = 'https://pyca.opencast.org'
+# Opencast username. Used for Http basic authentication.
+#name = "admin"
+# Password belonging to the above username
+# WARNING: When run in Opencast, this file is publicly accessibly
+#password = "opencast"
+
+### Settings of the metadata tab
+[metadata]
```

Comment:
The metadata and thumbnail tab were present before, but only as placeholders to show how the main menu would look like with more than two menu items. So enabling them by default will just show a placeholder message. Since they are A. supposed to be added in the future and B. supposed to be on/off configurable, I thought it simpler to turn them off through the config instead of commenting out a bunch of code in the frontend